ArduPlane/Parameters.cpp

```diff
@@ -1233,6 +1233,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("DSPOILER_AILMTCH", 21, ParametersG2, crow_flap_aileron_matching, 100),
+    
+    // @Param: ACRO_YAW_RATE
+    // @DisplayName: ACRO mode yaw rate, used by QACRO mode
+    // @Description: The maximum yaw rate at full stick deflection in ACRO mode
+    // @Units: deg/s
+    // @Range: 10 500
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("ACRO_YAW_RATE", 22, ParametersG2, acro_yaw_rate, 90),
```

Comment:
this should be in the Quadplane params

Reply:
I put it here because the roll and pitch rate parameters were here, and Plane also has an acro mode

Reply:
its for both acro and Qacro? if its only Q acro it should be in the Q params